### ISSUE TITLE
RGB Module- LED strip fails to turn off

### ISSUE THREAD

**Issue by FrankSchmidtDDS** [microsoft/jacdac#1363]
**Describe the bug**
A clear and concise description of what the bug is. When using RGB module and any number of LEDs the code block: "Set all led strip1 to black" does not turn all LEDs off. There is no : "set all LED strip1 to off".  

**To Reproduce**
1. Go to '...'  : this link  https://makecode.microbit.org/_912EvJ7v353e
2. Click on '....' Download to Microbit with Jacdac RGB module attached and neopixel strip or ring attached.
4. See error:  LEDs do not turn off when expected. 

**Expected behavior**
A clear and concise description of what you expected to happen. LEDs turn off with 'set all led to black' included in code, on button 1 pressed. 

**Screenshots**

**Desktop (please complete the following information):**
 - Browser [e.g. chrome, safari] :  Chrome, Edge, Brave

**Smartphone (please complete the following information):**

**Additional context**

**Comment by MKleinSB** [comment]
Yes, using another colour like yellow works. Black doesn´t work.

![grafik](https://github.com/microsoft/jacdac/assets/37311672/54d6eb2a-8491-4579-b5e3-7d77e936cc39)

**Comment by MKleinSB** [comment]
Turning off LEDs works in Software Jacdac! So perhaps it´s a problem with the transmission of 0x000000 in the Jacdac protocol?

**Comment by pelikhan** [comment]
@mmoskal didn't you fixed that one in jacdac-c ?
